### ISSUE TITLE
win_iis_website fails when removing a site

### ISSUE THREAD

**Issue by bgeihsgt** [ansible/ansible-modules-extras#1099]
##### Issue Type:

Bugfix Pull Request
##### Ansible Version:

```
ansible 2.0.0 (detached HEAD 5ae74f7d38) last updated 2015/10/13 06:57:44 (GMT +000)
  lib/ansible/modules/core: (detached HEAD d659c79db7) last updated 2015/10/13 06:57:49 (GMT +000)
  lib/ansible/modules/extras: (detached HEAD 0ff1776a84) last updated 2015/10/13 06:58:00 (GMT +000)
  config file = /vagrant/ansible/ansible.cfg
  configured module search path = /vagrant/ansible/library
```
##### Ansible Configuration:

I haven't changed anything that affects this module.
##### Environment:

Running from Ubuntu 14, managing Windows Server 2012 R2.
##### Summary:

win_iis_website fails with property error when removing a web site
##### Steps To Reproduce:
- Create an IIS website
- Run win_iis_website specifying that website to be absent
##### Expected Results:

It correctly removes the website and returns success with changed: true.
##### Actual Results:

It removes the website, but the win_iis_website module crashes because it complains that $site does not have a Name property.  I am submitting a pull request fixing this shortly.

**Comment by robynbergeron** [comment]
Hi @bgeihsgt -- thanks for submitting this bug report, and the accompanying PR to fix it -- #1100 :)

@whenrik -- thanks in advance for taking a look at this bug report and for your ongoing work in maintaining this module.

**Comment by robynbergeron** [comment]
[module: windows/win_iss_website.py]
